composer.json

```diff
@@ -26,12 +26,12 @@
         "doctrine/doctrine-bundle": "^1.12.3 || ^2.0",
         "doctrine/persistence": "^1.3.4 || ^2.0",
         "sonata-project/admin-bundle": "^3.99",
-        "sonata-project/block-bundle": "^3.20",
+        "sonata-project/block-bundle": "^4.0",
         "sonata-project/doctrine-extensions": "^1.8",
         "sonata-project/doctrine-orm-admin-bundle": "^3.19",
         "sonata-project/form-extensions": "^0.1.1 || ^1.4",
         "sonata-project/notification-bundle": "^3.8",
-        "sonata-project/seo-bundle": "^2.11",
```

Comment:
same

Reply:
Bc break might block to support both. And that's not our goal. In the same way we will bump sonata from 3 to 4, and it wont be possible to support both.